app/components/course-page/course-stage-step/first-stage-instructions-card/submit-code-step.hbs

```diff
@@ -4,7 +4,7 @@
   </p>
 </div>
 
-<CopyableTerminalCommand @commands={{array "git commit -am 'pass 1st stage' # any msg" "git push origin master"}} class="mb-3" />
+<CopyableTerminalCommand @commands={{array 'git commit -am "<any message>"' "git push origin master"}} class="mb-3" />
```

Comment:
@andy1li there's another variant of this for the first setup step that needs to be changed too: 

<img width="486" alt="Screenshot 2024-09-05 at 13 29 13" src="https://github.com/user-attachments/assets/23d45a98-a9d1-40ff-84c3-398054a6c2d9">

(Let's search by `git commit ` across the whole codebase and make sure everything is changed)